prereise/gather/demanddata/transportation_electrification/smart_charging.py

```diff
@@ -284,17 +289,31 @@ def smart_charging(
                 load_demand[i] + model_year_profile[i]
                 for i in range(day_iter * 24, (day_iter + 1) * 24)
             ]
+            adjusted_load += [load_demand[i] + model_year_profile[i] for i in range(48)]
+
+        elif day_iter == len(input_day) - 2:  # 364
+            adjusted_load = [
+                load_demand[i] + model_year_profile[i]
+                for i in range(day_iter * 24, ((day_iter + 1) * 24) + 24)
+            ]
             adjusted_load += [load_demand[i] + model_year_profile[i] for i in range(24)]
+
+        elif day_iter == len(input_day) - 3:  # 363
+            adjusted_load = [
+                load_demand[i] + model_year_profile[i]
+                for i in range(day_iter * 24, ((day_iter + 1) * 24) + 48)
+            ]
+
         else:
             adjusted_load = [
                 load_demand[i] + model_year_profile[i]
-                for i in range(day_iter * 24, day_iter * 24 + 48)
+                for i in range(day_iter * 24, day_iter * 24 + 72)
             ]
```

Comment:
I think we should be able to combine these branches together by following:
```python
day_num
for day_iter in range(day_num):
    adjusted_load = [
        load_demand[i] + model_year_profile[i]
        for i in range(day_iter * 24, day_iter * 24 + min(day_num - day_iter - 1, 3) * 24
    ]
    if 3 - day_num + day_iter > 0:
        adjusted_load += [load_demand[i] + model_year_profile[i] for i in range(24*(3 - day_num + day_iter))]
```

Reply:
Tried this out in the following branch:
https://github.com/Breakthrough-Energy/PreREISE/tree/load_branching_simplification
however I got the following test error:
<img width="1324" alt="Screen Shot 2022-10-05 at 12 16 51 PM" src="https://user-images.githubusercontent.com/1626883/194143913-484d314e-680c-4423-807b-3262fefd322a.png">

Reply:
Ah, my bad. I copied wrong line, now the test passes:
```python
    day_num = len(input_day)
    for day_iter in range(day_num):

        adjusted_load = [
            load_demand[i] + model_year_profile[i]
            for i in range(
                day_iter * 24, (day_iter + 1) * 24 + min(day_num - day_iter - 1, 2) * 24
            )
        ]

        if 3 - day_num + day_iter > 0:
            adjusted_load += [
                load_demand[i] + model_year_profile[i]
                for i in range(24 * (3 - day_num + day_iter))
            ]
```